spec/features/visitor_can_view_items_spec.rb

```diff
@@ -16,8 +16,12 @@
     within ".items" do
       expect(page).to have_content "Lucky Penny"
       expect(page).to have_content "$10.00"
+      expect(page).to have_css("img[alt='Lucky Penny']")
+      expect(page).to have_css("img[src=\"/images/Penny.png\"]")
```

Comment:
This ends up being redundant because of Factory Girl testing. After getting merged it will be(or need to be changed, actually) to something like "Lucky Item 1", and "http://bit.ly/1RlgZV5".